stubs/Markdown/markdown/inlinepatterns.pyi

```diff
@@ -53,31 +53,33 @@ class Pattern:
 class InlineProcessor(Pattern):
     safe_mode: bool
     def __init__(self, pattern: str, md: Markdown | None = None) -> None: ...
-    def handleMatch(self, m: re.Match[str], data) -> tuple[Element, int, int] | tuple[None, None, None]: ...  # type: ignore[override]
+    def handleMatch(self, m: re.Match[str], data: str) -> tuple[Element | str | None, int | None, int | None]: ...  # type: ignore[override]
```

Comment:
This should be `tuple[Element | str, int, int] | tuple[None, None, None]`. If one element is `None` then they must all be `None`.

Reply:
In reality *any* of them being None is understood.

https://github.com/Python-Markdown/markdown/blob/cef5afeb959d11e1ad5566fefa1f9a71136c1050/markdown/treeprocessors.py#L287

https://github.com/Python-Markdown/markdown/blob/cef5afeb959d11e1ad5566fefa1f9a71136c1050/markdown/treeprocessors.py#L304

Reply:
And consider these ones

https://github.com/Python-Markdown/markdown/blob/cef5afeb959d11e1ad5566fefa1f9a71136c1050/markdown/inlinepatterns.py#L891

https://github.com/Python-Markdown/markdown/blob/cef5afeb959d11e1ad5566fefa1f9a71136c1050/markdown/inlinepatterns.py#L355

But I don't know, maybe it's for the best to keep this signature stricter